ArchiSteamFarm/Bot.cs

```diff
@@ -554,6 +567,43 @@ internal sealed class Bot : IDisposable {
 			return result;
 		}
 
+		internal async Task<Dictionary<string, string>> GetKeysFromFile(string filePath) {
+			if (string.IsNullOrEmpty(filePath) || !File.Exists(filePath)) {
```

Comment:
`!File.Exists(filePath)` is not non-intended - you should instead either add another condition below without logging (just return `null`) or keep it like that but add proper logic for skipping non-existing files in the caller. Also make it `private` because no other code accesses it.

Reply:
IMHO when the file does not exist it should return an empty Dictionary but this is discussable so i'll stick with null, as you would

Reply:
Empty dictionary in json output would suggest that file exists but is empty, not that it doesn't exist.

Reply:
Well... we return null if something went wrong whereas a missing file is not necessarily wrong but only means there are no keys present which would be the same as an empty file (which asf will never create)

Reply:
ASF is not the only process in the world permitted to read/write those files, users can also screw them up. Although in this case returning empty dictionary is valid when file doesn't exist, so you can use it if you want to.